courses/fundamentals_of_ada/135_visibility.rst

```diff
@@ -312,12 +312,14 @@ Examples
 .. code:: Ada
 
    package P is
-     type T1 is range 1 .. 10;
-     type T2 is range 1 .. 10;
-     type T3 is range 1 .. 10;
+     type Miles_T is digits 6;
+     type Hours_T is digits 6;
+     type Speed_T is digits 6;
      -- "use type" on any of T1, T2, T3
```

Comment:
`-- "use type" on any of Miles_T, Hours_T, Speed_T`

Reply:
Done